oklayoutinflator/src/main/java/tech/okcredit/layout_inflator/OkLayoutInflater.kt

```diff
@@ -33,17 +39,17 @@ class OkLayoutInflater(context: Context) {
             val view = inflateView(resId, parent)
             withContext(Dispatchers.Main) { callback.invoke(view) }
         }
-    }
 
-    fun cancel() {
-        coroutineContext.cancelChildren()
+
```

Comment:
How can auto cancellation be used in custom views? Many items in the recycler view are inflated asynchronously. Canceling occurs on onDetachedFromWindow. How does it work here?

Reply:
onDetachedFromWindow is tied with onDestroy.There  will always be parent context which can be observed .I have tested on the detech method check once with logs or toast.